### ISSUE TITLE
VPS usage

### ISSUE THREAD

**Issue by cosminzm** [Ezzpify/HourBoostr#219]
Can it be used on a vps?
To stop consuming resources from the PC !!

**Comment by Messiah93** [comment]
Yes... Both Windows and Linux. For Linux, use Mono as CLI (EG mono HourBoostr.exe).
